lib/binding_web/prefix.js

```diff
@@ -1,6 +1,6 @@
 (function (root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    define([], factory);
+  if (typeof define === 'function' && define.amd) { 
+    define([], factory);                            
```

Comment:
I don't think you meant to include those whitespaces

Reply:
Good catch 👍